src/components/Services.jsx

```diff
@@ -47,4 +70,9 @@ const mapStateToProps = state => {
   }
 }
```

Comment:
Can you add proptypes ? 

Also having a TODO note to remove the need of this selector and use a CC's query can be great, no?

Reply:
I can add PropTypes, but I haven't touched the `mapStatesToProps` in this PR… I also don't think we need a TODO for that?

Reply:
OK